hibernate5/src/main/java/com/baeldung/hibernate/config/Suppliers.hbm.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<!-- Generated Feb 9, 2017 11:31:36 AM by Hibernate Tools 5.1.0.Final -->
+<hibernate-mapping>
+    <class name="com.baeldung.hibernate.pojo.Suppliers" table="suppliers" optimistic-lock="version">
```

Comment:
@KevinGilmore We have used entity classes mapping so this hbm is no more required. Earlier I tried it with hbm and thus this code was from that trial. We dont need this anymore and have removed in today's commits.